services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -895,7 +895,9 @@ private boolean canViewNews(String authenticatedUser, Space space) throws Except
   }
 
   private boolean canScheduleNews(String authenticatedUser, Space space) {
-    return spaceService.isManager(space, authenticatedUser) || spaceService.isRedactor(space, authenticatedUser);
+    org.exoplatform.services.security.Identity currentIdentity = ConversationState.getCurrent().getIdentity();
+    return spaceService.isManager(space, authenticatedUser) || spaceService.isRedactor(space, authenticatedUser) || currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, PUBLISHER_MEMBERSHIP_NAME) ||
+           currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, "*");
```

Comment:
```suggestion
    return spaceService.isManager(space, authenticatedUser) || spaceService.isRedactor(space, authenticatedUser) || currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, PUBLISHER_MEMBERSHIP_NAME);
```
no need of currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, "*") the above condition is enough IMHO